tick-api/src/main/java/me/aurium/tick/driver/TickDriver.java

```diff
@@ -0,0 +1,54 @@
+package me.aurium.tick.driver;
+
+import me.aurium.tick.container.container.JDBCContainer;
+
+import java.sql.*;
+import java.util.Properties;
+import java.util.Set;
+import java.util.logging.Logger;
+
+public class TickDriver implements Driver {
```

Comment:
Are you sure you need to implement java.sql.Driver? There are a lot of things which could go wrong with doing so which might present behavior incompatibilities between TickDriver and the actual underlying JDBC driver.

Reply:
This only exists because testcontainers does something similar: it will route connections with driver jdbc:tick:<identifier> to the identified managed container.